sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -363,6 +363,12 @@ abstract class OffsetWindowFunction
    */
   val direction: SortDirection
 
+  /**
+   * Whether the offset is start with the current row. If `startWithCurrentRow` is false, `offset`
+   * means the offset is start with the first row of the entire window partition.
+   */
```

Comment:
let's document how it interacts with `direction`

Reply:
OK.